src/Native/System.Native/pal_process.cpp

```diff
@@ -11,6 +11,7 @@
 #include <limits.h>
 #include <signal.h>
 #include <stdlib.h>
+#include <sys/param.h>
```

Comment:
This is for the check `#ifdef BSD`:
`sys/param.h` is available in all POSIX and `BSD` macro is only defined in BSD-backed OSes.